Zenodo.js

```diff
@@ -114,38 +116,79 @@ function doWeb(doc, url) {
 	}
 }
 
+var zoteroType = {
+	"figure": "artwork",
+	 "article": "report"
+}
```

Comment:
see above -- I'm leaving that as document on purpose (though feel free to disagree